shop/src/main/java/ro/msg/learning/shop/configuration/ClassConfig.java

```diff
@@ -0,0 +1,24 @@
+package ro.msg.learning.shop.configuration;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import ro.msg.learning.shop.service.strategy.LocationStrategy;
+import ro.msg.learning.shop.service.strategy.SingleLocationStrategy;
+
+@Configuration
+public class ClassConfig {
```

Comment:
Here you could use a more suggestive naming for the class like LocationStrategyConfig.